src/components/CardHome.module.css

```diff
@@ -0,0 +1,32 @@
+.cardContainer .button {
+  background-color: rgb(243, 195, 0);
+  color: rgb(0, 62, 82);
+  border: none;
+}
+
+.cardContainer .carding {
+  display: flex;
+  justify-content: center;
+  background-color: rgb(41, 143, 168);
+  color: white;
+  padding-bottom: 1%;
```

Comment:
I'm not sure it is considered good practice to use percentages for margin and padding.  px or rem/em is usually better, especially for something like a card